lua/orgmode/org/mappings.lua

```diff
@@ -783,6 +787,13 @@ function OrgMappings:open_at_point()
     return vim.cmd(string.format('edit %s', url))
   end
 
+  if url:find('^custom:') then
```

Comment:
If emacs org-mode uses `shell` then we should stick to that. The idea is that a `.org` file that works in emacs should also work in nvim-orgmode.

Reply:
Changed it to `shell:` as requested.